test/connectors/EmailVerificationConnectorSpec.scala

```diff
@@ -24,12 +24,6 @@ class EmailVerificationConnectorSpec extends ControllerBaseSpec {
   lazy val connector = new EmailVerificationConnector(mock[HttpClient], mockConfig)
 
   "EmailVerificationConnector" should {
-
-    "generate the correct verify check URL" in {
```

Comment:
Do you still need to have a test for this function for coverage?

Reply:
That's a good point i'll remove it in both